app/components/AppearanceSettings.vue.ts

```diff
@@ -36,14 +33,17 @@ export default class AppearanceSettings extends Vue {
   }
 
   openFFZSettings() {
-    this.windowsService.createOneOffWindow({
-      componentName: 'FFZSettings',
-      title: $t('FrankerFaceZ Settings'),
-      queryParams: {},
-      size: {
-        width: 800,
-        height: 800
-      }
-    }, 'ffz-settings');
+    this.windowsService.createOneOffWindow(
+      {
```

Comment:
this is an example of a change i don't really like, imo it just adds lines unnecessarily and just looks kinda gross but its probably not configurable :/

Reply:
Yeah this is one of those where I agree but could live with either option.  Neither option significantly impacts readability IMO.  @blackxored do you know which rule this is by any chance?

Reply:
I do not believe this one is configurable. I'm ambivalent about it as well, I guess there's a small point to be made about having the argument on its line (similar to multiline i.e, easy add/removal). But other than that and cosmetics I'm not sure. 
Oh, and it reads better if you're into parameter hints 😂 

![capture](https://user-images.githubusercontent.com/133308/50121680-5d528e80-020e-11e9-81a9-9186f566d446.PNG)

Reply:
K.  I'm assuming we can all live with this.  If anyone disagree LMK, but for now this one is resolved.